cmd/install/assets/hypershift_operator.go

```diff
@@ -496,6 +496,28 @@ func (o HyperShiftOperatorDeployment) Build() *appsv1.Deployment {
 					},
 					PriorityClassName:  HypershiftOperatorPriortyClass,
 					ServiceAccountName: o.ServiceAccount.Name,
+					InitContainers: []corev1.Container{
+						{
+							Name:            "init-environment",
+							Image:           image,
+							ImagePullPolicy: corev1.PullIfNotPresent,
+							Command:         []string{"/usr/bin/hypershift-operator"},
+							Args:            []string{"init"},
+							// needed since hypershift operator runs with anyuuid scc
```

Comment:
can you elaborate why we need this here?

Reply:
Ah sorry, I didn't take time to understand it or verify. It looks like the hypershift operator runs with scc `nonroot-v2` which requires only the UID to be set on container manifests ([src](https://docs.openshift.com/container-platform/4.12/authentication/managing-security-context-constraints.html#default-sccs_configuring-internal-oauth)). 

```
$ oc describe po -n hypershift operator-fcbf86449-5hhpq
Name:                 operator-fcbf86449-5hhpq
Namespace:            hypershift
Labels:               app=operator
...
                      openshift.io/scc: nonroot-v2
```

I've removed the rest of the security context for this container and have only left the uid. 

Follow-up question:
1. Do you know where/how the `scc` is set for this deployment?